test/test_events.py

```diff
@@ -0,0 +1,229 @@
+"""
+Tests for the events module.
+"""
+from datetime import datetime
+from test.conftest import MockUQCSBot, TEST_CHANNEL_ID
+from pytz import timezone
+from unittest.mock import patch
+from uqcsbot.utils.itee_seminar_utils import (HttpException, get_seminars)
+
+NO_EVENTS_MESSAGE = "_There don't appear to be any events in the next *2* weeks_\r\n" \
+    "For a full list of events, visit: https://uqcs.org.au/calendar.html " \
+    "and https://www.itee.uq.edu.au/seminar-list"
+
+
+def mocked_html_summary_get_typical():
+    """
+    Returns locally stored HTML that represents a typical seminar listing.
+    """
+    f = open("test/ITEE_Upcoming_Seminars.html", "r")
```

Comment:
Nitpicking (probably too much 2002 marking), but maybe a variable name like html_data, or just return the chained function calls. Not super worried about whether this/others like this are actioned.